evm/vm/evm.py

```diff
@@ -243,30 +236,42 @@ def import_block(self, block):
         except BlockNotFound:
             raise ValidationError("Parent ({0}) of block {1} not found".format(
                 block.header.parent_hash, block.header.hash))
+
         init_header = self.create_header_from_parent(parent_header)
         parent_evm = type(self)(self.db, init_header)
         imported_block = parent_evm.get_vm().import_block(block)
```

Comment:
These two lines make me think this should be a method.  Something like `EVM.get_ancestor_evm(....)` or `EVM.get_parent_evm(...)`.  Hides some of the grossness of creating a temporary header and then using `type(self)` to derive the class.  Maybe it can be a `classmethod` to have the class directly available.

I'm not even remotely sure this is a good idea.  Use your best judgement.